src/__tests__/attributes.js

```diff
@@ -352,10 +352,28 @@ test('insensitive attribute selector 2', '[href=TEsT i  ]', (t, tree) => {
 test('insensitive attribute selector 3', '[href=test i]', (t, tree) => {
     t.deepEqual(tree.nodes[0].nodes[0].value, 'test');
     t.deepEqual(tree.nodes[0].nodes[0].insensitive, true);
+
+    tree.nodes[0].nodes[0].insensitive = false;
+
+    t.deepEqual(tree.toString(), '[href=test ]');
 });
 test('capitalized insensitive attribute selector 3', '[href=test I]', (t, tree) => {
     t.deepEqual(tree.nodes[0].nodes[0].value, 'test');
     t.deepEqual(tree.nodes[0].nodes[0].insensitive, true);
+
+    tree.nodes[0].nodes[0].insensitive = false;
+
+    t.deepEqual(tree.toString(), '[href=test ]');
+});
+
+test('insensitive attribute selector 4', '[href="test"i]', (t, tree) => {
+    t.deepEqual(tree.nodes[0].nodes[0].value, 'test');
+    t.deepEqual(tree.nodes[0].nodes[0].insensitive, true);
+});
+
+test('capitalized insensitive attribute selector 4', '[href="test"I]', (t, tree) => {
+    t.deepEqual(tree.nodes[0].nodes[0].value, 'test');
+    t.deepEqual(tree.nodes[0].nodes[0].insensitive, true);
```

Comment:
Just for tests, let's add:
- `[href="test" i ]`
- `[href=test i ]`

Reply:
done!